client/app/components/services/service-explorer/service-explorer.component.js

```diff
@@ -40,7 +40,7 @@ function ComponentController($state, ServicesState, Language, ListView, Chargeba
       viewType: 'listView',
       cardConfig: getCardConfig(),
       listConfig: getListConfig(),
-      listActions: getListActions(),
+      listActions: {},
```

Comment:
I believe this to be a symptom the problem, not the underlying issue.  A fix shouldn't mess with this component.